src/Components/SmartComponents/Reports/Executive/__snapshots__/FirstPage.test.js.snap

```diff
@@ -48,6 +48,10 @@ exports[`FirstPage component Should match snapshot 1`] = `
       <FirstPage
         data={
           {
+            "systemCountPerType": {
```

Comment:
If you have added a test for FirstPage in RTL, do we really need the snapshot testing ? 
This is debatable of course, but from the top of my mind 😃

Reply:
This is needed to make sure that the counts are passed down to the FirstPage component.